hw/drivers/flash/spiflash/include/spiflash/spiflash.h

```diff
@@ -25,17 +25,24 @@
 #endif
 #include <hal/hal_flash_int.h>
 #include <hal/hal_spi.h>
+#if MYNEWT_VAL(BUS_DRIVER_PRESENT)
+#include <bus/spi.h>
```

Comment:
you could rebase this on top of https://github.com/apache/mynewt-core/pull/1568/ and just include `bus/drivers/spi_common.h` here and also add dependency to `pkg.yml`:
```
pkg.deps.BUS_DRIVER_PRESENT:
    - "@apache-mynewt-core/hw/bus/drivers/spi_common"
```